CHANGELOG.md

```diff
@@ -7,6 +7,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.8.4] - 2020-03-24
+
```

Comment:
We should make a separate PR to release that thing. Then also the links at the bottom need sorting. Since I want to get my own changes out I would do the release once this PR is merged. 

```suggestion
```